doc/source/ray-air/doc_code/accelerate_trainer.py

```diff
@@ -0,0 +1,73 @@
+import torch
+import torch.nn as nn
+
+from accelerate import Accelerator
+
+import ray
+from ray.air import session, Checkpoint
+from ray.train.huggingface.accelerate import AccelerateTrainer
+from ray.air.config import ScalingConfig
+
+
+# If using GPUs, set this to True.
+use_gpu = False
+
+
+input_size = 1
+layer_size = 15
+output_size = 1
+num_epochs = 3
+
+
+class NeuralNetwork(nn.Module):
+    def __init__(self):
+        super(NeuralNetwork, self).__init__()
+        self.layer1 = nn.Linear(input_size, layer_size)
+        self.relu = nn.ReLU()
+        self.layer2 = nn.Linear(layer_size, output_size)
+
+    def forward(self, input):
+        return self.layer2(self.relu(self.layer1(input)))
+
+
+def train_loop_per_worker():
+    accelerator = Accelerator()
+    dataset_shard = session.get_dataset_shard("train")
+    model = NeuralNetwork()
+    loss_fn = nn.MSELoss()
+    optimizer = torch.optim.SGD(model.parameters(), lr=0.1)
+
+    model, optimizer = accelerator.prepare(model, optimizer)
+
+    for epoch in range(num_epochs):
+        for batches in dataset_shard.iter_torch_batches(
+            batch_size=32, dtypes=torch.float
+        ):
+            inputs, labels = torch.unsqueeze(batches["x"], 1), batches["y"]
+            output = model(inputs)
+            loss = loss_fn(output, labels)
+            optimizer.zero_grad()
+            accelerator.backward(loss)
+            optimizer.step()
+            print(f"epoch: {epoch}, loss: {loss.item()}")
+
+        session.report(
+            {},
```

Comment:
can you add comments so we know what this empty dict is?
it's metrics dict right? maybe it's better to send loss as a metric

Reply:
this is the same as in `doc/source/ray-air/doc_code/torch_trainer.py` and `doc/source/ray-air/doc_code/hvd_trainer.py`

Reply:
wait, why do we want to carry "unclear" stuff into all new examples?